setup.py

```diff
@@ -10,6 +11,24 @@
     'pyyaml<7',
 ]
 
+extras = {
+    "develop": [
+        "black<25",
+        "mypy<1.10",
+        "poethepoet<0.26",
+        "pyproject-fmt<1.8",
+        "ruff<0.5",
+        "validate-pyproject<0.17",
+    ],
+    "test": [
+        "grafana-dashboard==0.1.1",
+        "pydantic<2",
```

Comment:
The `pydantic` dependency may have slipped in, and can probably be removed.

Reply:
No idea where that comes from...
Maybe I have duplicated a setup.py from an another tool without knowing what I do... (that is really possible!)
Feel free to remove it is not required.

Reply:
I think it slipped in on my updates, and the PR comment was merely a note to self. Thanks!

Reply:
I found out why it was needed: Pydantic 2.x croaks on Python 3.12, due to how it is used by `grafana-dashboard`.